README.md

```diff
@@ -1,70 +1,170 @@
-# Getting Started with Create React App
+<a name="readme-top"></a>
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="./assets/image/book-icon.png" alt="logo" width="140"  height="auto" />
```

Comment:
- [ ] Your logo is not displaying, please make sure the image is upload to GitHub and that it is properly linked  👍🏾

Reply:
@wayungi Thanks for your review :) would you please help me to solve the error when app runs? I checked it and rerun the application on my machine. I didn't see any error. The app is running in the browser perfectly. If you like, then I can send you a video on it. Thanks again :)